<?xml version="1.0"?>
<launch>
  <node name="slam_gmapping" pkg="gmapping" type="slam_gmapping">
    <remap from="/scan1" to="/Diff_Drive/laser/scan_multi"/>
    <param name="base_frame" value="chassis"/>
  </node>
</launch>
